system-test/spanner.ts

```diff
@@ -1253,7 +1262,8 @@ describe('Spanner', () => {
         err => {
           assert.ifError(err);
 
-          database.run('SELECT * FROM Singers', (err, rows) => {
+          // tslint:disable-next-line: no-any
+          database.run('SELECT * FROM Singers', (err, rows: any) => {
```

Comment:
There's actually a [`Row` interface](https://github.com/googleapis/nodejs-spanner/blob/master/src/partial-result-stream.ts#L62) you can use here.

Reply:
I need to change type of this as `any` b/c not able to find the property like `SingerId` and `Name` is used below.

Reply:
I think this is because we're casting this incorrectly. Further down we map the rows into JSON objects but cast it as `Row[]` which a `Row` is an array, not an object. I believe creating a new variable with the correct type should circumvent the need to cast to any.